src/sentry_plugins/asana/client.py

```diff
@@ -14,12 +14,12 @@ def get_issue(self, issue_id):
         return self.get("/tasks/%s" % issue_id)
 
     def create_issue(self, workspace, data):
-        asana_data = {"name": data["title"], "notes": data["description"], "workspace": workspace}
+        asana_data = {"name": data["title"], "notes": data["description"], "workspace": str(workspace)}
         if data.get("project"):
-            asana_data["projects"] = [data["project"]]
+            asana_data["projects"] = [str(data["project"])]
```

Comment:
You should use `six.text_type()` instead of `str()`. `str()` will throw an exception in python2 when non-ascii bytes are encountered.